admin-dev/themes/new-theme/js/pages/order/view.js

```diff
@@ -159,9 +163,19 @@ $(() => {
 
   function handleUpdateOrderStatusButton() {
     const $btn = $(OrderViewPageMap.updateOrderStatusActionBtn);
+    const $wrapper = $(OrderViewPageMap.updateOrderStatusActionInputWrapper);
 
-    $(OrderViewPageMap.updateOrderStatusActionInput).on('change', (event) => {
-      const selectedOrderStatusId = $(event.currentTarget).val();
+    $wrapper.click(() => {
+      $wrapper.toggleClass('select-status-opened');
+    });
+
+    $(OrderViewPageMap.updateOrderStatusActionInput).on('change', event => {
+      const $element = $(event.currentTarget);
+      const $option = $('option:selected', $element);
+      const selectedOrderStatusId = $element.val();
+
+      $wrapper.css('background-color', $option.data('background-color'));
+      $wrapper.toggleClass('is-bright', $option.data('is-bright') !== undefined);
```

Comment:
```suggestion
      $wrapper
        .css('background-color', $option.data('background-color'))
        .toggleClass('is-bright', $option.data('is-bright') !== undefined);
```

Reply:
It's less than 100 character, eslint will inline it after mergin it on develop